jamf_protect/manifest.json

```
@@ -0,0 +1,35 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a863cfe8-5ba4-45ae-923c-d273510f099c",
+  "app_id": "jamf-protect",
+  "display_on_public_website": false,
```

```suggestion
  "display_on_public_website": true,
```

We use this field to display the integrations in the docs (for instance [this page](https://docs.datadoghq.com/integrations/kafka/?tab=host) for kafka)